CONTRIBUTING.md

```diff
@@ -1,18 +1,26 @@
 # Contributing to Dogecoin Core
 
-The Dogecoin Core project operates an open contributor model where anyone is
-welcome to contribute towards development in the form of peer review, testing
-and patches. This document explains the practical process and guidelines for
-contributing.
+Dogecoin Core is an open source project, and we would welcome contributions which provably
+improve the state of the software. As a pointer on where to start, please look at the
+[projects](https://github.com/dogecoin/dogecoin/projects) currently underway,
+where we track issues which are being worked on or need attention from others.
 
-Firstly in terms of structure, there is no particular concept of "Core
-developers" in the sense of privileged people. Open source often naturally
-revolves around meritocracy where longer term contributors gain more trust from
-the developer community. However, some hierarchy is necessary for practical
-purposes. As such there are repository "maintainers" who are responsible for
-merging pull requests as well as a "lead maintainer" who is responsible for the
-release cycle, overall merging, moderation and appointment of maintainers.
+## Branch Strategy
 
+Dogecoin Core's default branch is intentionally a stable release, so that anyone downloading the code and compiling it gets a stable release.
+Active development occurs on branches named after the version they are targetting. Currently there are:
+
+* [Next minor - 1.14.4](https://github.com/dogecoin/dogecoin/tree/1.14.4-dev)
```

Comment:
I'm not sure if it's a good idea to include explicitly current  development.
Will need to be updated regularly.

Reply:
Removed and replaced with a written example instead.